.github/workflows/test.yml

```diff
@@ -24,6 +24,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     env:
       APP_VERSION: ${{ matrix.app_version }}
+      NEOVIM_VERSION: ${{ matrix.app_version == 'stable' && 'stable' || 'v0.10.0' }}
```

Comment:
Good call. Indeed legacy is 0.10.0 and stable is 0.10.1 and will soon be 0.11.0 I believe